cmd/tools/grafana/dashboard_test.go

```diff
@@ -781,19 +781,11 @@ func checkExemplarIsFalse(t *testing.T, path string, data []byte) {
 	}
 }
 
-var uidExceptions = map[string]bool{
-	"cmode-details/volumeBySVM.json":    true,
-	"cmode-details/volumeDeepDive.json": true,
-}
-
-func checkUIDIsBlank(t *testing.T, path string, data []byte) {
+func checkUIDNonEmpty(t *testing.T, path string, data []byte) {
```

Comment:
minor, but checkUIDNotEmpty might be better

Reply:
done